Update installing-a-github-app-from-github-marketplace-for-your-organ… (#34063)
github/docs: content/apps/using-github-apps/installing-a-github-app-from-github-marketplace-for-your-organizations.md

The "app manager" role in an organization does not give a person the ability to install a {% data variables.product.prodname_github_app %} in the organization. For more information, see "[AUTOTITLE](/apps/maintaining-github-apps/about-github-app-managers)."

> For enterprise managed user accounts, only enterprise owners can purchase and set up paid a {% data variables.product.prodname_github_app %} for an organizations within their enterprise. Enterprise members cannot purchase a paid {% data variables.product.prodname_github_app %}. Organization owners with an enterprise managed user account can still install a free {% data variables.product.prodname_github_app %}.

Reviewer: @D1M1TR10S In case you missed it, I think we should make the above suggested changes 🙏🏻

> For enterprise managed user accounts, only enterprise owners can purchase and install a paid {% data variables.product.prodname_github_app %} for an organization within their enterprise. Enterprise members cannot purchase a paid {% data variables.product.prodname_github_app %}. Organization owners with an enterprise managed user account can still install a free {% data variables.product.prodname_github_app %}.

Reviewer 2: This will version the content so that it only shows up for the GHEC versions of the docs. This aligns with the "[About Enterprise Managed Users](https://docs.github.com/en/enterprise-cloud@latest/admin/managing-iam/understanding-iam-for-enterprises/about-enterprise-managed-users)" article version. 

```suggestion
{% ifversion ghec %}

For enterprise managed user accounts, only enterprise owners can purchase and install a paid {% data variables.product.prodname_github_app %} for an organization in the enterprise. Enterprise members cannot purchase a paid {% data variables.product.prodname_github_app %}. Organization owners with an enterprise managed user account can still install a free {% data variables.product.prodname_github_app %}.

{% endif %}
```

Reviewer 3: @D1M1TR10S Hello! 👋 I'm going to go ahead and commit @jc-clark's suggestion - then we'll get this merged 💛
